books/free-programming-books-ar.md

```diff
@@ -68,6 +68,7 @@
 
 * [تعلم JavaScript&rlm;](https://itwadi.com/node/3002) - Cody Lindley, عبداللطيف ايمش (PDF)
 * [سلسلة تعلم Next.js بالعربية](https://blog.abdelhadi.org/learn-nextjs-in-arabic/) - Flavio Copes, عبدالهادي الأندلسي
+* [Eloquent Javascript (hsoub)&rlm;](https://academy.hsoub.com/files/27-%D8%A7%D9%84%D8%A8%D8%B1%D9%85%D8%AC%D8%A9-%D8%A8%D9%84%D8%BA%D8%A9-%D8%AC%D8%A7%D9%81%D8%A7%D8%B3%D9%83%D8%B1%D8%A8%D8%AA/) - Marijn Haverbeke (PDF/articles)
```

Comment:
```suggestion
* [Eloquent Javascript (hsoub)&rlm;](https://academy.hsoub.com/files/27-%D8%A7%D9%84%D8%A8%D8%B1%D9%85%D8%AC%D8%A9-%D8%A8%D9%84%D8%BA%D8%A9-%D8%AC%D8%A7%D9%81%D8%A7%D8%B3%D9%83%D8%B1%D8%A8%D8%AA/) - Marijn Haverbeke (PDF/articles)
```
We try to take the title from the source, in this case the source uses a translated title "البرمجة بلغة جافاسكربت"
It should just be (PDF).
"hsoub" can be credited for the translation after the original author as done in the entry below - it doesn't belong in the title.
We prefer that you modify your PR instead of closing it and making a new one - that way we can see what your changes are.